### PR TITLE
Feature/GAT-1338: Add METADATA QUALITY score to search results

### PR DESCRIPTION
**Pull request by kymmeh**

### CODE REVIEW

**Review by kandaj — APPROVED** [review]

File: src/components/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import ActionBarStatus from './ActionBarStatus';
```
Comment (ThinkingMonkeyLtd) [inline src/components/index.js:1]: Great!

File: src/components/QualityScore/QualityScore.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    const IconComponent = iconMappings[rating];
+
+    if (!IconComponent) return null;
+
+    return (
+        <ToolTip
+            placement='bottom'
+            text={
+                <span
+                    // eslint-disable-next-line react/no-danger
```
Comment (ThinkingMonkeyLtd) [inline src/components/QualityScore/QualityScore.jsx:32]: Does it need // eslint-disable-next-line react/no-danger
                    dangerouslySetInnerHTML

File: src/pages/dataset/DatasetPage.js [hunk 3]
```diff
@@ -941,7 +875,12 @@ class DatasetDetail extends Component {
                                             </span>
                                         </Col>
                                         <Col xs={4} md={2} className='text-right'>
-                                            <Metadata />
+                                            <QualityScore
+                                                rating={data.datasetfields.metadataquality.weighted_quality_rating}
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dataset/DatasetPage.js:879]: Could we expand if it isn't a pain

File: src/components/QualityScore/QualityScore.jsx (hunk 2 above)
Comment (kymmeh) [reply src/components/QualityScore/QualityScore.jsx:32]: I've just tested and without it it won't render the HTML tags.

File: src/pages/dataset/DatasetPage.js (hunk 3 above)
Comment (kymmeh) [reply src/pages/dataset/DatasetPage.js:879]: Do you mean spread the props?

Comment (kymmeh) [reply src/pages/dataset/DatasetPage.js:879]: I was trying to avoid having the underscore syntax within our components.

Comment (kymmeh) [reply src/pages/dataset/DatasetPage.js]: Discussed and destructured 👍

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by kandaj — APPROVED** [review]

**Pull request merged**